src/command/identity/export.ts

```diff
@@ -0,0 +1,77 @@
+import { Argument, LeafCommand } from 'furious-commander'
+import { exit } from 'process'
+import { Identity, IdentityType } from '../../service/identity/types'
+import { RootCommand } from '../root-command'
+import { printNoIdentitiesError, promptUserForIdentity } from './common'
+
+export class Export extends RootCommand implements LeafCommand {
+  // CLI FIELDS
+
+  public readonly name = 'export'
+
+  public readonly description = 'Export identity'
+
+  @Argument({ key: 'identity-name', describe: 'Name of the identity to be exported' })
```

Comment:
If this argument is required (as opposed to optional) you need to pass in a `required: true` property. Then it will be written in the help as required and also the framework will complain if the user does not provide it.

Reply:
This is purposely not required - it works the same way as the `identity remove` command. We show an option picker with V3 identities only where the end user can select an identity in a user friendly way. Do you agree with this approach?